packages/metals-vscode/src/testExplorer/analyzeTestRun.ts

```diff
@@ -37,7 +37,7 @@ export function analyzeTestRun(
     if (result != null) {
       // if suite run contains test cases (run was started for (single) test case)
       if (kind === "testcase") {
-        analyzeTestCases(run, result, [test], test.parent);
+        analyzeTestCases(run, result, [test], test._metalsParent);
```

Comment:
Is this change needed? The type hasn't changes, so this seems unnecessary

Reply:
Yeah, but this did not work. The tests did not cover this, since the `id` and DAP results were both encoded in the same way (both had parent name in id) in tests, which according to a comment in the code doesn't have to be true in practice. But when I changed the encoding this part started to matter.